tests/setup_workdir.sh

```diff
@@ -81,6 +81,13 @@ else
 fi
 
 WORKDIR=$1
+
+# Fetch PXC versions
+MAJOR_VERSION=`git ls-remote --refs --sort='version:refname' --tags https://github.com/percona/percona-xtradb-cluster | \
```

Comment:
To avoid confusion. I think it is better we rename 

`MAJOR_VERSION` => `LATEST_VERSION`
`MINOR_VERSION` => `VERSION_SUFFIX`

Reply:
Ok